src/main/target/TINYFISH/target.mk

```diff
@@ -0,0 +1,10 @@
+F3_TARGETS  += $(TARGET)
+FEATURES    = VCP ONBOARDFLASH
+
+TARGET_SRC = \
+            drivers/accgyro_mpu.c \
+            drivers/flash_m25p16.c \
+            drivers/accgyro_spi_mpu6000.c \
+            drivers/light_ws2811strip.c \
```

Comment:
You don't need to include the `drivers/light_*` files, since they are included by default for F3 targets and above.